pandas/core/sparse/series.py

```diff
@@ -604,14 +604,38 @@ def sparse_reindex(self, new_index):
 
     def take(self, indices, axis=0, convert=True, *args, **kwargs):
         """
-        Sparse-compatible version of ndarray.take
```

Comment:
use a shared doc string for all public ``.take()``

Reply:
Done.